libRocket.cmake

```diff
@@ -0,0 +1,549 @@
+# Create libRocket.
```

Comment:
THe librocket repo has the actual build file. I felt this was much simpler than doing External project and hardcoding library paths. This is geared towards unvanquished (omitting python stuff).